site/guide/manage-users.qmd

```diff
@@ -0,0 +1,69 @@
+---
+title: "Manage users"
+keywords: "manage users, user management, ValidMind"
+---
+
+Managing users is fundamental to maintaining access control and efficient user management in the ValidMind system. This task topic will guide you through the process of inviting new users, assigning them to specific groups, and configuring their roles. Whether you're a group administrator or a superuser, understanding these steps is crucial for ensuring the right individuals have the appropriate access and permissions within the system.
+
+## Viewing your account profile
+1. Log in to the ValidMind UI.
+
+2. Go to Settings at the sidebar and select  **Profile** at Your Account.
+
+3. You can find the following at your profile page.
+   - Profile details 
+   - Roles attached to your account
+   - Values for API Key and Secret Key
+
+## Prerequisites
+
+To perform these next steps, you must be a group administrator or a superuser.
+
+## Invite new users
+
+1. [Log in to the ValidMind UI]({{< var vm_ui >}}).
+
+2. Click **Settings**. 
+
+   - Navigate to **User Permissions** in the sidebar. 
```

Comment:
Since this bulleted list is quite specific about which UI elements you interact with and the sequence you need to follow, it's likely better off as a lettered list. Bullets don't specify a required order but a., b., etc., does.